org/lateralgm/subframes/FontFrame.java

```diff
@@ -183,6 +186,16 @@ public void focusLost(FocusEvent e)
 		rangeList.setLayoutOrientation(JList.VERTICAL);
 		rangeList.setVisibleRowCount(6);
 
+		rangeList.getInputMap().put(KeyStroke.getKeyStroke(VK_DELETE, 0), "delete selected");
```

Comment:
We've actually extracted all keyboard shortcuts to a properties file. The intention was to later allow users to change keyboard shortcuts IDE-wide. It's ok, I'll amend this before merging.
https://github.com/IsmAvatar/LateralGM/blob/master/org/lateralgm/messages/keyboard.properties